modules/twinklexfd.js

```diff
@@ -520,6 +579,7 @@ Twinkle.xfd.callbacks = {
 		if (form.xfdtarget2) {
 			params.target2 = form.xfdtarget2.value;
 		}
+		params.delsort_cats = $(form.delsort).val();
```

Comment:
I know this will likely be dealt with in another of your pulls, but why the jquery here compared to the others?

Reply:
I had thought this would be nullified by #597 but seems not.  Still think this should match the others.

Reply:
Can't do that, `value` property for a multi-select input gives only the first selected option rather than the list of all selected options, which we need.

Reply:
Indeed!